packages/reporter/src/header/DebugDismiss.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import DebuggerIcon from '-!react-svg-loader!@packages/frontend-shared/src/assets/icons/debugger_x16.svg'
+import DeleteIcon from '-!react-svg-loader!@packages/frontend-shared/src/assets/icons/status-failed_x12.svg'
+import events from '../lib/events'
+
+export const DebugDismiss = (props: { matched: number, total: number }) => {
+  return (
+    <button className="debug-dismiss" onClick={() => events.emit('testFilter:cloudDebug:dismiss')}>
+      <DebuggerIcon />
+      <span>{props.matched} / {props.total} tests</span>
```

Comment:
Pluralization edge case: Only one test in the project and you've filtered to include it - `1 / 1 tests`